frontends/tf-frontend/tf_mlir_ext/transforms/rewrite_to_custom_call.cc

```diff
@@ -450,15 +450,31 @@ struct RewriteOneHot : public OpRewritePattern<TF::OneHotOp> {
 // Repeat Pattern
 //===----------------------------------------------------------------------===//
 struct RewriteRepeat : public RewritePattern {
-  RewriteRepeat(MLIRContext *context, PatternBenefit benefits = 1)
-      : RewritePattern("tf.Repeat", benefits, context) {}
+  RewriteRepeat(MLIRContext *context, PatternBenefit benefits = 1,
+                int64_t oBatchSize = -1)
+      : RewritePattern("tf.Repeat", benefits, context),
+        outBatchSize(oBatchSize) {}
```

Comment:
Firstly, I think `set-shape` should be split from `convert-op-to-custom-call`.   
Secondly, applying `set-shape` on tf dialect may be more suitable. Because it should working with `tf-shape-infer`.

Reply:
（BTW， I think shape-inference should be completely solved on tf dialect. `tf-to-mhlo` should only be lowering.

Reply:
I fully agree with your idea. The set shape of the op of tf should be separated into a pass for the tf dialect to solve this problem. In order not to hinder other students' work, I will do this work at the beginning of next month

Reply:
ok